lib/applications.js

```diff
@@ -347,3 +347,34 @@ exports.exportAll = async (req, res) => {
 
     return res.send(resultBuffer);
 };
+
+exports.getStats = async (req, res) => {
+    const statsObject = {
+        by_event: [],
+        by_body: [],
+        by_nationality: []
+    };
+
+    const applications = await Application.findAll({ attributes: ['event_id', 'body_name', 'nationality'] });
+
+    statsObject.by_event = helpers
+        .countByField(applications, 'event_id');
+
+    for (let i = 0; i < statsObject.by_event.length; i++) {
```

Comment:
this would do quite a lot of requests, can optimize it here by preloading all events at once

Reply:
Good one, I should have improved it now